picard/mbjson.py

```diff
@@ -192,6 +194,16 @@ def _translate_artist_node(node):
     if config.setting['translate_artist_names']:
         locale = config.setting["artist_locale"]
         lang = locale.split("_")[0]
+
+        if config.setting['translate_artist_names_exception']:
+            char_set_exception = config.setting["artist_locale_exception"]
+
+            ad = alphabet_detector.AlphabetDetector()
+            char_sets = ad.detect_alphabet(node['name'])
+
+            if char_set_exception in char_sets:
```

Comment:
Since `char_set_exception` is only used here, I think we can get rid of it:

```python
            if config.setting["artist_locale_exception"] in char_sets:
```

And if we go for a simple method:

```python
    if (config.setting['translate_artist_names_exception'] and 
        config.setting["artist_locale_exception"] in detect_alphabet(node['name'])):
        return node['name'], node['sort-name']
```

Reply:
This simplification is subjective, for example for me the form I used is simpler. Anyway done.